lib/ui/painting/color_filter.h

```diff
@@ -31,9 +31,11 @@ class ColorFilter : public RefCountedDartWrappable<ColorFilter> {
   // (translate). skia is normalized, treating the last column as 0...1, so we
   // post-scale here before calling the skia factory.
   static sk_sp<SkColorFilter> MakeColorMatrixFilter255(const float array[20]);
+  static sk_sp<SkColorFilter> MakeHSLAMatrixFilter255(const float array[20]);
 
   void initMode(int color, int blend_mode);
   void initMatrix(const tonic::Float32List& color_matrix);
+  void initHSLAMatrix(const tonic::Float32List& color_matrix);
```

Comment:
This will need corresponding changes in painting.dart. It will also need changes on the web side (/cc @ferhatb @yjbanov) to support it. 

This will also need tests. See for example https://github.com/flutter/engine/blob/master/testing/dart/color_filter_test.dart